zipkin-lens/index.html

```diff
@@ -16,6 +16,7 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <base href="/zipkin" >
```

Comment:
without this, anything that emulates zipkin-server's ZIPKIN_UI_BASE_PATH handling would have broke.